### PR TITLE
Fix/find project url state

### LINKED ISSUE #735: Find Projects: Navigating to url with more than two filters in a category fails to apply filters
If one filters by more than filter in a category, the resulting url will fail to navigate with the filters applied.

## Repro Steps ##

1. Navigate to Find Projects page
2. Filter by at least two filters in the same category
3. Copy resulting url, and navigate to that url in a new tab

Expected: The url navigates to the Find Projects page with the filters applied
Actual: The url navigates to the Find Projects page but the filters are not applied

Code Location:
https://github.com/DemocracyLab/CivicTechExchange/blob/8cca934ab802a715dcf68cae5419b22f3e333e8c/common/components/controllers/FindProjectsController.jsx#L20

### PR DESCRIPTION
**Pull request by KaueReinbold**
- Deccode the search url to property load the filter selected
- On the first load check for tag selected to expand the menu
 
closes #735

### CODE REVIEW

File: common/components/componentsBySection/FindProjects/Filters/RenderFilterCategory.jsx [hunk 1]
```diff
@@ -34,11 +34,14 @@ class RenderFilterCategory<T> extends React.PureComponent<Props<T>, State> {
     let cs = _.concat(c, s);
     //cs is now an array of each key we want to use for expand/collapse tracking
 
-    //make an object and push in each key from cs, set all values to false
+    //make an object and push in each key from cs, set all values based wether the tag inside is checked
```
Comment (marlonkeating) [inline common/components/componentsBySection/FindProjects/Filters/RenderFilterCategory.jsx:37]:
```suggestion
    //make an object and push in each key from cs, set all values based whether the tag inside is checked
```

File: common/components/componentsBySection/FindProjects/Filters/RenderFilterCategory.jsx [hunk 2]
```diff
@@ -34,11 +34,14 @@ class RenderFilterCategory<T> extends React.PureComponent<Props<T>, State> {
...
     const collector = {};
     for (var key in cs) {
       let val = cs[key];
-      collector[val] = false;
+      collector[val] = props.data.some(tag => 
+        props.checkEnabled(tag) && 
+        [tag.category, tag.subcategory].includes(val) 
+      );
```
Comment (marlonkeating) [inline common/components/componentsBySection/FindProjects/Filters/RenderFilterCategory.jsx:44]: We use _.some and _.includes in the project, to get around certain browser incompatibilities.

**Review by marlonkeating — CHANGES_REQUESTED** [review]

File: common/components/componentsBySection/FindProjects/Filters/RenderFilterCategory.jsx (hunk 2 above)
Comment (KaueReinbold) [reply common/components/componentsBySection/FindProjects/Filters/RenderFilterCategory.jsx]: Good to know, just sent the change!

File: common/components/componentsBySection/FindProjects/Filters/RenderFilterCategory.jsx (hunk 1 above)
Comment (marlonkeating) [reply common/components/componentsBySection/FindProjects/Filters/RenderFilterCategory.jsx:37]: The typo reappeared.  If you make a text commit via github, make sure you merge the commit into your code branch before making further changes.

**Review by marlonkeating — CHANGES_REQUESTED** [review]

File: common/components/componentsBySection/FindProjects/Filters/RenderFilterCategory.jsx (hunk 1 above)
Comment (KaueReinbold) [reply common/components/componentsBySection/FindProjects/Filters/RenderFilterCategory.jsx]: just sent the adjust on the comment, sorry about that

**Review by marlonkeating — APPROVED** [review]

**Pull request merged**